docs/source/catalog.rst

```diff
@@ -311,6 +311,33 @@ The "types" of caching are that supported are listed in ``intake.source.cache.re
 the docstrings of each for specific parameters that should appear in the cache block.
 
 
+Compressed Files
+----------------
```

Comment:
Sorry, I think this should be `'''''''''`, else it'll be a higher-level heading than intended

Reply:
ok, no worries